docs/reference/eql/functions.asciidoc

```diff
@@ -223,6 +224,72 @@ field datatypes:
 *Returns:* boolean or `null`
 ====
 
+[discrete]
+[[eql-fn-stringcontains]]
+=== `stringContains`
+
+Returns `true` if a source string contains a provided substring.
+
+[%collapsible]
+====
+*Example*
+[source,eql]
+----
+// process.command_line = "start regsvr32.exe"
+stringContains(process.command_line, "regsvr32")  // returns true
+stringContains(process.command_line, "start ")    // returns true
+stringContains(process.command_line, "explorer")  // returns false
+
+// process.command_line = [ "start regsvr32.exe", "start explorer.exe" ]
+stringContains(process.command_line, "regsvr32")  // returns false
+stringContains(process.command_line, "explorer")  // returns true
+
+// empty strings
+stringContains("", "")                            // returns false
+stringContains("start regsvr32.exe", "")          // returns false
+
+// null handling
+stringContains(null, "regsvr32")                  // returns null
```

Comment:
Based off the conversation today, `null` in any required input will lead to a `null` output
```suggestion
stringContains(null, "regsvr32")                  // returns null
stringContains(process.command_line, null)  // returns null
```

Reply:
👍 Updated with 8bcb8f8f4afbc2edc48530396a028acbc4f30e36.